scene/resources/packed_scene.cpp

```diff
@@ -465,27 +441,20 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 
 	for (const DeferredNodePathProperties &dnp : deferred_node_paths) {
 		// Replace properties stored as NodePaths with actual Nodes.
-		Node *other = dnp.base->get_node_or_null(dnp.path);
-
-		const String string_property = dnp.property;
-		if (string_property.contains("/indices/")) {
-			// For properties with "/indices/", the replacement takes place inside an Array.
-			const String base_property = string_property.get_slice("/", 0);
-			const int index = string_property.get_slice("/", 2).to_int();
+		if (dnp.value.get_type() == Variant::ARRAY) {
+			Array paths = dnp.value;
 
 			bool valid;
-			Array array = dnp.base->get(base_property, &valid);
+			Array array = dnp.base->get(dnp.property, &valid);
```

Comment:
Just realised something this should be duplicated first. This was not present previously and is causing another kind of bug where the default value gets modified instead of anew instance. Since You're modifying I felt like it could be included here directly but I can make a separate PR if needed.

Reply:
Put the duplication below after the validity check

Reply:
yeah that was a good idea

Reply:
Yeah this is right we are never using the value we are getting from there. What still needs to be acounted for is the typing of the array but since we know for sure this is a `Array[Node]` this should be easy. This might also fix the disappearrence because in fact if the value was invalid it would never set the array which would make Nodes dissapear. 
Overall I think this is a really good idea.

Reply:
I'll leave it the way it is for now and if desired we can modify it later, unless it's considered significant here